src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -271,6 +272,23 @@ private Task<LocationItem> CreateItemHomeAsync()
 			return CreateSectionAsync(SectionType.Home);
 		}
 
+		private void CreateItemSettingsAsync()
+		{
+			var item = new LocationItem()
+			{
+				Text = "Settings".GetLocalizedResource(),
+				Path = "Settings",
+				Icon = new BitmapImage(new Uri(Constants.FluentIconsPaths.HomeIcon)),
```

Comment:
Is there a way to use an animated icon?

Reply:
Yeah.
I’m gonna add a FrameworkElement instead of a IconElement to render PathIcons and OpacityIcons. Is there a new type of icons?

Reply:
AnimatedIcon is what we use for the existing settings button.

Reply:
Ah true, sure.